src/main/java/it/eng/idsa/businesslogic/processor/producer/ProducerSendDataToBusinessLogicProcessor.java

```diff
@@ -10,6 +10,7 @@
 import java.util.regex.Matcher;
```

Comment:
Don't format th all content of the class because after you format all content of the class it is hard to see what is changed. For example, you changed few rows, but it is shown like you changed all content in the class.

Reply:
In these cases we should share the same formatting rules.